Plugin~/WebRTCPlugin/Context.cpp

```diff
@@ -8,17 +8,16 @@
 namespace WebRTC
 {
     ContextManager ContextManager::s_instance;
-    bool ContextManager::s_use_software_encoder = false;
 
-    Context* ContextManager::GetContext(int uid)
+    Context* ContextManager::GetContext(int uid, UnityEncoderType encoderType)
```

Comment:
Adding encoderType as a parameter here seems a bit strange because this function is supposed to return an existing context. 

If we want to create a default Context when it doesn't exist, I think it's better to just hardcode it somewhere rather than using a parameter.

Reply:
Sure, I guess we should define a method to create context instance like below
`ContextManager::CreateContext(int uid, UnityEncoderType encoderType)`

I will change the behavior of `GetContext` so it returns null if `context` instance does not exist.